src/main/java/com/eva/logic/commands/AddApplicationCommand.java

```diff
@@ -23,6 +28,8 @@
             + "Example: " + COMMAND_WORD + " "
             + "src\\main\\java\\com\\eva\\logic\\parser\\resume.txt \n"
             + "or if you would like to scan a sample resume: addapplication 1 sample";
+    public static final String MESSAGE_WRONG_PANEL = "Please switch to applicant list panel "
+            + "via 'list a-' to add application of applicant";
```

Comment:
Since they're able to add an application from a profile perhaps you should include that in the message as well? Maybelike "Please switch to applicant list panel (list a-) or view applicant profile panel to add application to applicant."